apps/emqx_authn/src/simple_authn/emqx_authn_http.erl

```diff
@@ -266,10 +268,10 @@ generate_request(Credential, #{method := Method,
     Body = replace_placeholders(Body0, Credential),
     case Method of
         get ->
-            NPath = append_query(Path, BaseQuery ++ Body),
+            NPath = append_query(Path, Credential, BaseQuery ++ Body),
```

Comment:
Please change to the following way? Otherwise, KVs in body will be processed repeatedly: 

```
BaseQuery = replace_placeholders(BaseQuery0, Credential),
NPath = append_query(Path, Credential, BaseQuery ++ Body),
```